src/data/cassandra/impl/Statement.hpp

```diff
@@ -119,6 +119,9 @@ class Statement : public ManagedObject<CassStatement> {
             // reinterpret_cast is needed here :'(
             auto const rc = bindBytes(reinterpret_cast<unsigned char const*>(value.data()), value.size());
             throwErrorIfNeeded(rc, "Bind string (as bytes)");
+        } else if constexpr (std::is_convertible_v<DecayedType, Text>) {
+            auto const rc = cass_statement_bind_string(*this, idx, value.text.c_str());
```

Comment:
This is not an issue now but will be later for sure - there probably is `cass_statement_bind_string_n` where you can pass the size of the string as last arg

Reply:
I will fix it now. I think string data() makes sure the data() ends with '\0' after C++11. But I can not find it in reference.